cinema_service/settings.py

```diff
@@ -39,13 +39,15 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
```

Comment:
Consider wrapping the inclusion of `debug_toolbar` in `INSTALLED_APPS` with a condition to check if `DEBUG` is `True`. This will prevent it from being included in a production environment.